concent_api/conductor/tests/test_integration_verification.py

```diff
@@ -0,0 +1,290 @@
+            reverse(
+                'conductor:report_upload',
+                kwargs = {
+                    'file_path': file_path
+                }
+            ),
+            content_type = 'application/octet-stream',
+        )
+
+        self.assertEqual(response.status_code,         200)
+        self.assertEqual(UploadReport.objects.count(), 1)
+
+        upload_report = UploadReport.objects.first()
+        self.assertEqual(upload_report.path,           file_path)
+        self.assertEqual(upload_report.upload_request, upload_request)
+
+    def test_conductor_should_create_upload_report_and_do_not_link_to_unrelated_upload_request(self):
+        file_path = 'blender/result/ef0dc1/ef0dc1.zzz523.zip'
+
+        verification_request = VerificationRequest(
+            subtask_id = '1'
+        )
+        verification_request.full_clean()
+        verification_request.save()
+
+        upload_request = UploadRequest(
+            path                    = 'blender/result/ef0dc1/ef0dc1.zzz523.arj',
+            verification_request    = verification_request,
+        )
+        upload_request.full_clean()
+        upload_request.save()
+
+        response = self.client.get(
+            reverse(
+                'conductor:report_upload',
+                kwargs = {
+                    'file_path': file_path
+                }
+            ),
+            content_type = 'application/octet-stream',
+        )
+
+        self.assertEqual(response.status_code,         200)
+        self.assertEqual(UploadReport.objects.count(), 1)
+
+        upload_report = UploadReport.objects.first()
+        self.assertEqual(upload_report.path,           file_path)
+        self.assertEqual(upload_report.upload_request, None)
+
+    @override_settings(CELERY_ALWAYS_EAGER = True)
+    def test_conductor_should_schedule_verification_order_task_if_all_related_upload_requests_have_reports(self):
+        file_path = 'blender/result/ef0dc1/ef0dc1.zzz523.zip'
+
+        verification_request = VerificationRequest(
+            subtask_id = '1'
+        )
+        verification_request.full_clean()
+        verification_request.save()
+
+        upload_request = UploadRequest(
+            path                    = file_path,
+            verification_request    = verification_request,
+        )
+        upload_request.full_clean()
+        upload_request.save()
+
+        response = self.client.get(
+            reverse(
+                'conductor:report_upload',
+                kwargs = {
+                    'file_path': file_path
+                }
+            ),
+            content_type = 'application/octet-stream',
+        )
+
+        self.assertEqual(response.status_code,         200)
+        self.assertEqual(UploadReport.objects.count(), 1)
+
+        upload_report = UploadReport.objects.first()
+        self.assertEqual(upload_report.path,           file_path)
+        self.assertEqual(upload_report.upload_request, upload_request)
```

Comment:
please add `TODO` to not forget adding additional asserts, once `verification_order_task` is implemented

Reply:
What additional asserts ?

Reply:
test is called `test_conductor_should_schedule_verification_order_task_if_all_related_upload_requests_have_reports` and I do not see anything about scheduling verification order task. Should it be that way?

Reply:
Added :+1: